shenyu-admin/src/main/java/org/apache/shenyu/admin/model/query/MockRequestRecordQuery.java

```diff
@@ -0,0 +1,201 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.shenyu.admin.model.query;
+
+import java.io.Serializable;
+import java.util.Objects;
+import org.apache.shenyu.admin.model.page.PageParameter;
+
+/**
+ * this is mock request record query.
+ */
+public class MockRequestRecordQuery implements Serializable {
+
+    private static final long serialVersionUID = 6736947701814601503L;
+
+    /**
+     * the api id.
+     */
+    private String apiId;
+
+    /**
+     * the request host.
+     */
+    private String host;
+
+    /**
+     * the request url.
+     */
+    private String url;
+
+    /**
+     * the request param in url.
+     */
+    private String pathVariable;
+
+    /**
+     * the request param after url.
+     */
+    private String query;
+
```

Comment:
Add `header` condition. As for `body`, I don't think MySQL is suitable for it, but ES is too complex, so ignore `body`.